site/docs/reference/Connectors/capture-connectors/SQLServer/sqlserver-batch.md

```diff
@@ -1,81 +1,180 @@
-executing queries and translating the results into JSON documents.
+This connector captures data from SQL Server databases into Flow collections by
+periodically executing queries and translating the results into JSON documents.
 
-For local development or open-source workflows, [`ghcr.io/estuary/source-sqlserver-batch:dev`](https://ghcr.io/estuary/source-sqlserver-batch:dev) provides the latest version of the connector as a Docker image. You can also follow the link in your browser to see past image versions.
+## When to use this connector
 
-We recommend using our [SQL Server CDC Connector](./sqlserver.md) instead
-if possible. Using CDC provides lower latency data capture, delete and update events, and usually
-has a smaller impact on the source database.
+We recommend using our [SQL Server CDC Connector](http://go.estuary.dev/source-sqlserver)
+instead when possible. CDC provides lower latency data capture, delete and
+update events, and typically has a smaller impact on the source database.
 
-However there are some circumstances where this might not be feasible. Perhaps you need
-to capture from a managed SQL Server instance which doesn't support logical replication.
-Or perhaps you need to capture the contents of a view or the result of an ad-hoc query.
-That's the sort of situation this connector is intended for.
+However, the batch connector is the right choice when:
 
-The number one caveat you need to be aware of when using this connector is that **it will
-periodically execute its update query over and over**. For example, if you set the polling interval to
-5 minutes, a naive `SELECT * FROM foo` query against a 100 MiB view will produce 30 GiB/day
-of ingested data, most of it duplicated. The default polling interval is set
-to 24 hours to minimize the impact of this behavior, but even then it could mean a lot of
-duplicated data being processed depending on the size of your tables.
+- Your SQL Server instance doesn't support Change Tracking (e.g., some managed services)
+- You need to capture from database views
+- You want to execute ad-hoc or custom queries
+- You need to capture from a read replica that doesn't have Change Tracking enabled
 
-If you start editing these queries or manually adding capture bindings for views or to run
-ad-hoc queries, you need to either have some way of restricting the query to "just the new
-rows since last time" or else have your polling interval set high enough that the data rate
-`<DatasetSize> / <PollingInterval>` is an amount of data you're willing to deal with.
+## Supported versions and platforms
 
-## Configuration
+This connector works with all supported versions of SQL Server on major cloud platforms
+(including Amazon RDS, Azure SQL Database, and other managed services), as well as
+self-hosted instances.
+
+:::tip Configuration Tip
+To capture data from databases hosted on your internal network, you must
+use [SSH tunneling](/guides/connect-network/).
```

Comment:
Same suggestion as mysql batch, to mention private cloud networking for private deployments